### PR TITLE
Extraced fonts from Google to be self hosted

### PR DESCRIPTION
**Pull request by networkException**
- Changes to font loading and implementation
 - Added $accent as a color variable

Google Fonts is, as of right now, still not conform with the GDPR. Furthermore, in _my option_, a visitor of the Wiki should be certain, that none of their information reaches third parties, which at least a log on Googles servers may contain. _This is definitely not legal advice_ and the changes should be discussed further in comments.

### CODE REVIEW

File: docs/stylesheets/dark-theme.scss [hunk 1]
```diff
@@ -1,9 +1,59 @@
...
+         url('./fonts/roboto-mono-v7-cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin-regular.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
 
+body, input {
+    font-family: "Roboto", "Helvetica Neue", Helvetica, Arial, sans-serif
+}
+
+code, kbd, pre {
+    font-family: "Roboto Mono", "Courier New", Courier, monospace
+}
```
Comment (J0B10) [inline docs/stylesheets/dark-theme.scss]: I suggest moving all that font related stuff to a separate file. Just create a new `fonts.css` inside the `stylesheets` dir.  
That way this file only contains color theme related content.

You'll then need to add that stylesheet to the [`extra_css` section inside the mkdocs.yml](https://github.com/codeoverflow-org/chatoverflow-wiki/blob/f019b7e900b70925e13966d67130623a0f274c77/mkdocs.yml#L36-L38).

File: docs/stylesheets/dark-theme.scss [hunk 2]
```diff
@@ -1,9 +1,59 @@
...
+$navbar_header_background: $accent;
+
+/* Extraced fonts from Google */
+/* roboto-regular - cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin */
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Roboto'), local('Roboto-Regular'),
+       url('./fonts/roboto-v20-cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin-regular.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
```
Comment (J0B10) [inline docs/stylesheets/dark-theme.scss]: Could we move the fonts to `docs/fonts` instead of `docs/stylesheets/fonts`? 

Guess that makes more sense...

**Review by J0B10 — CHANGES_REQUESTED** [review]
You're totally right. I should have thought of that.

I requested a few changes for a better repository structure but nothing big.

**Comment by J0B10** [comment]
And we need to include the license and copyright notice to comply with Apache 2.0.

But I can do this when i merge the PR.

**Comment by networkException** [comment]
I moved the files as suggested

**Review by J0B10 — APPROVED** [review]

**Pull request merged**